### DISCUSSION TITLE
Percent of Billed and Per Diem updates - Mandatory?

### DISCUSSION THREAD

**Discussion by dawndrake58** [CMSgov/price-transparency-guide#412] [Q&A] [answered]
Are these additional pricing methodologies required or optional for the 7/1/22 implementation?  The additional negotiated rate types require changes at the highest level of the files. The time remaining does not allow for full development and testing of the changes. for complex contract pricing.

**Comment by ryan-thomas-peterson** [comment]
Adding onto this question - If a provider contract is based on percentage of billed charges and a chargemaster rate for the provider can be reasonably estimated, is the plan obligated to list the percentage?  Or could they list the dollar amount equivalent?

**Reply by Data-san** [reply]
Any idea when per diem was added to the schema? Appears to be post March 1st.

**Reply by tdfow** [reply]
it was added with v0.11.0 (end of Feb/1st of March), as v0.11.0 through v1.0 -- was published all together to the master.

**Reply by tdfow** [reply]
@ryan-thomas-peterson if you are posting a rate versus a percentage, then the negotiation type would be 'derived' or 'negotiated' instead of 'percentage'.   The definition of the use of 'percentage' would always have the percentage in the rate field.

**Reply by ryan-thomas-peterson** [reply]
@tdfow , thanks we'll likely use derived for these absent other guidance.  Our organizational view is that if these files are supposed to help entrepreneurs and layer into the 1/1/23 requirements that supplying percentages are likely to make things much harder to support the rest of the law and some of the underlying intent.

**Reply by Data-san** [reply]
Thanks. Related discussion didn't cover per diem, and percentage was committed Feb 28th. It would be best for CMS to avoid such late changes when development has already started (per their encouragement),

**Reply by bettinavanover** [reply]
@shaselton-usds, can you please confirm whether or not "percentage" (and additional pricing methodologies) is required or optional for the 7/1/22 implementation?

**Comment by shaselton-usds** [comment, marked as answer]
If you have negotiated arrangements based on either percentage of billed charges or per diem, then those negotiation types must be used.  While these files can help with 1/1/23 tool requirements, the In-Network file is first and foremost meant to reflect the contracts and not a retrospective look (i.e. using historic claims to come up with a dollar amount) on what the contract could result in.

@dawndrake58 The hope with allowing support of disclosing a percentage vs a dollar amount for percentage of billed arrangements was to lessen the work needed by looking at historical claims or coming up with time frames in which to look back at historical claims. By moving requirements closer to the contract language, additional analysis can be avoided to derive dollar amounts.
